README.md

```diff
@@ -98,6 +98,8 @@ PyCM is the swiss-army knife of confusion matrices, targeted mainly at data scie
 
 ⚠️  PyCM 2.4 is the last version to support **Python 2.7** & **Python 3.4**
 
+⚠️  Plotting capability require **Matplotlib (>= 3.0.0)** or **Seaborn (>= 0.9.1)**   
```

Comment:
Please change it to: 

⚠️  Plotting capability requires **Matplotlib (>= 3.0.0)** or **Seaborn (>= 0.9.1)**